config.json

```diff
@@ -1230,6 +1242,11 @@
       "uuid": "6d0806a3-e881-48fc-960a-d7663de0667a",
       "slug": "numbers",
       "name": "Numbers"
+    },
+    {
+      "uuid": "8afaf3b9-5910-4d6f-a35d-aaf9439381f6",
+      "slug": "booleans",
+      "name": "Booleans"
```

Comment:
This should not be added in this PR, as it is part of adding the concept, not the exercise.

Reply:
Understood. I added only for the config let tests. I will move Annalyn's Infiltration to a practice exercise and eliminate this.

Reply:
Removed.